apps/bare-expo/ios/Podfile.lock

```diff
@@ -633,7 +633,7 @@ PODS:
     - React
   - react-native-viewpager (5.0.11):
     - React-Core
-  - react-native-webview (11.13.0):
```

Comment:
Not sure why webview is being updated, will check tomorrow!

Reply:
it looks like we didn't run pod install in bare-expo after merging #15330